styles.css

```diff
@@ -781,30 +766,25 @@ a {
 /* Mobile popup section */
 
 .modal {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
   position: fixed;
   left: 0;
   top: 0;
   backdrop-filter: blur(5px);
   background-color: rgba(0, 0, 0, 0.6);
-  height: 100%;
   width: 100%;
+  height: 100%;
+  z-index: 1000;
+  overflow-y: hidden;
+  padding: 1rem;
```

Comment:
- [ ] Kindly consider changing the `overflow-y: hidden` to `overflow-y: scroll` such that users are able to scroll to see the hidden content of the model on small screen sizes to create a good user experience. Currently, when your model is opened on a small screen size, part of the content hides from the user and they're unable to scroll to see it because you have the `overflow-y: hidden`. Kindly take a look at the image below for more details:

Your design:
<img width="373" alt="Screenshot 2023-04-20 at 00 15 43" src="https://user-images.githubusercontent.com/107192076/233225874-684f1ffc-cefe-484d-b243-8d198e5e9b14.png">